waterlog/src/actions/SegmentActions.js

```diff
@@ -0,0 +1,12 @@
+import {FETCH_SEGMENTS} from "./types";
+ 
+  export const fetchSegments = () => dispatch => {   
+    fetch('https://localhost:44382/api/segments')//Change to use either localhost/server
+      .then(res => res.json())
+      .then(segments =>
```

Comment:
add error handling if the call fails for whatever reason. Ideally dispatch an error action and then in the reducer set the state to some default state to prevent the app from crashing